README.md

```diff
@@ -123,20 +180,28 @@ amioChat.messages.send({
 })
 ```
 
-### messages.sendText(text)
+### messages.sendText(text, metadata)
```

Comment:
We should extract the `contentBuilder` to a separate lib and use it here (as you once said already). https://github.com/amio-io/amio-sdk-js#api---content-builders---horizontal-scroll

We would then have only `messages.send(message)` just like in our SDK. I would postpone this issue until next sprint.